src/esp/geo/VoxelGrid.h

```diff
@@ -0,0 +1,392 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#ifndef ESP_GEO_VOXEL_GRID_H_
+#define ESP_GEO_VOXEL_GRID_H_
+
+#include <vector>
+
+#include <Corrade/Containers/ArrayViewStl.h>
+#include <Corrade/Containers/StridedArrayView.h>
+#include <Corrade/Utility/Directory.h>
+#include <Magnum/GL/PixelFormat.h>
+#include <Magnum/Image.h>
+#include <Magnum/Math/Color.h>
+#include <Magnum/Math/CubicHermite.h>
+#include <Magnum/Math/Range.h>
+#include <Magnum/PixelFormat.h>
+
+#include "esp/assets/MeshData.h"
+#include "esp/assets/MeshMetaData.h"
+#include "esp/core/esp.h"
+#include "esp/geo/geo.h"
+#include "esp/gfx/Drawable.h"
+#include "esp/gfx/magnum.h"
+
+#ifdef ESP_BUILD_WITH_VHACD
+#include "VHACD.h"
+#endif
+
+namespace Mn = Magnum;
+namespace Cr = Corrade;
```

Comment:
Hm, AFAIK we so far avoided these in the headers and used the full names to avoid alias pollution, only doing this in cpp files.

Opinions? @aclegg3

Reply:
Yeah, that makes sense to me.